internal/wasm/jit/engine.go

```diff
@@ -288,7 +293,7 @@ const (
 
 // jitCallStatusCode represents the result of `jitcall`.
 // This is set by the jitted native code.
-type jitCallStatusCode byte
+type jitCallStatusCode uint32
```

Comment:
changed a byte to uint32 so that there's no ambiguity in the offset (by Go's alignment adjustment)